chainer/optimizers/ada_grad.py

```diff
@@ -20,7 +20,7 @@ class AdaGradHyperparameter(tpe.Protocol):
 
 _default_hyperparam = optimizer.Hyperparameter()  # type: AdaGradHyperparameter # NOQA
 _default_hyperparam.lr = 0.001
-_default_hyperparam.eps = 1e-8
+_default_hyperparam.batch_size_factor = 1
```

Comment:
I think the default `batch_size_factor` can be set in `optimizer.Hyperparameter()` base class.
`eps` should not be removed.